test/acceptance/client.js

```diff
@@ -38,7 +38,8 @@ describe('The node.js OC client', function(){
   var getRegExpFromJson = function(x){
     return JSON.stringify(x)
       .replace(/\+/g, '\\+')
-      .replace(/\[/g, '\\[');
+      .replace(/\[/g, '\\[')
+      .replace(/\?\?/g, '\\?'); //In our json regexp in order to preserve a single ? we are escaping it  with ??
```

Comment:
@matteofigus, I did some changes and introduced new tests to cover the expectations we have for get-component-data to prepare a GET request when just one component is passed and for href-builder to include the component parameters as URL query params for the GET request URL.
However I can't come up with an idea for getting rid of this '??' escaping. Perhaps I don't understand your question entirely :-(